src/IterableCodeExtractor.php

```diff
@@ -172,16 +159,16 @@ function ( $file, $key, $iterator ) use ( $include, $exclude, $extensions ) {
 					/** @var RecursiveCallbackFilterIterator $iterator */
 					/** @var SplFileInfo $file */
 
-					if ( static::isExcluded( $file, $exclude ) && ( empty( $include ) || ! static::isIncluded( $file, $include ) ) ) {
-						return false;
+					if ( $iterator->hasChildren() ) {
```

Comment:
We might need to make this smarter, to avoid traversing the entire `node_modules` for example, when there's no rule that "includes" anything related to that.